lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -644,4 +644,29 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
       end)
     end)
   end
+
+  defp normalize_path_env(env_vars) when is_list(env_vars) do
+    env_vars |> Enum.into(%{}) |> normalize_path_env()
+  end
+
+  defp normalize_path_env(env_vars) when is_map(env_vars) do
+    {path, env_vars} = Map.pop(env_vars, "PATH")
+
+    if path do
+      {os_path, separator} =
+        case :os.type() do
+          {:win32, _} -> {:os.cmd('echo %PATH%'), ";"}
+          _ -> {:os.cmd('echo $PATH'), ":"}
+        end
```

Comment:
We can just do `System.get_env("PATH", "")` to access the path, right? So I would store the initial value on init, then:

* when path env is set we prepend to the one in state
* when path env is deleted we restore the one from state

Reply:
Yeah, tbh, this works much better